runtime/compiler/runtime/SymbolValidationManager.cpp

```diff
@@ -92,7 +92,17 @@ TR::SymbolValidationManager::defineGuaranteedID(void *symbol)
    {
    uint16_t id = getNewSymbolID();
    _symbolToIdMap.insert(std::make_pair(symbol, id));
-   _idToSymbolsMap.insert(std::make_pair(id, symbol));
+   setSymbolOfID(id, symbol);
+   }
+
+void
+TR::SymbolValidationManager::setSymbolOfID(uint16_t id, void *symbol)
+   {
+   if (id >= _idToSymbolTable.size())
+      _idToSymbolTable.resize(id + 1, NULL);
```

Comment:
To clarify, do the elements (that haven't already been set) get initialized to `0`?

Reply:
Yes, the second argument (`NULL` in this case) is the value to copy into new elements when `resize` increases the size of the vector.